scripts/generate-infra.sh

```diff
@@ -8,12 +8,8 @@ infra_name="${1:-small_infra}"
 
 echo "Loading $infra_name"
 python3 python/railjson_generator/railjson_generator/scripts/generate.py /tmp/generated_infras/
-docker cp "/tmp/generated_infras/${infra_name}/infra.json" osrd-api:/tmp/infra.json
-docker exec osrd-api python manage.py import_railjson "${infra_name}" /tmp/infra.json
+docker cp "/tmp/generated_infras/${infra_name}/infra.json" osrd-editoast:/tmp/infra.json
+docker exec osrd-editoast editoast import-railjson "${infra_name}" /tmp/infra.json
 
 echo "Generate layers"
 docker exec osrd-editoast editoast generate
-
-echo "Loading example rolling stock"
-docker cp python/api/static/example_rolling_stock.json osrd-api:/tmp/stock.json
-docker exec osrd-api python manage.py import_rolling_stock -f /tmp/stock.json
```

Comment:
When is the example rolling stock loaded now? I believe the readme instructions should be updated to account for the change

Reply:
Editoast doesn't support a command line to import a rolling stock.
I can update the readme and indicate that a rolling stock must be created using the editor. Is it satisfying?

Reply:
Well that's a problem, as users surely want to try OSRD with some example data